assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.js

```diff
@@ -26,18 +26,20 @@ import { sprintf, __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
-import Button from '../Button';
-import Checkbox from '../Checkbox';
-import BannerNotification from '../notifications/BannerNotification';
-import ProgressBar from '../ProgressBar';
-import Spinner from '../Spinner';
+import { CORE_MODULES } from '../../../googlesitekit/modules/datastore/constants';
+import Button from '../../Button';
+import Checkbox from '../../Checkbox';
+import BannerNotification from '../../notifications/BannerNotification';
+import ProgressBar from '../../ProgressBar';
+import Spinner from '../../Spinner';
+import Error from './Error';
 
 const { useDispatch, useSelect } = Data;
 
 export default function ModuleRecoveryAlert() {
 	const [ checkboxes, setCheckboxes ] = useState( null );
 	const [ recoveringModules, setRecoveringModules ] = useState( false );
+	const [ error, setError ] = useState( null );
```

Comment:
As mentioned before, let's source this from the datastore instead.

Reply:
Updated, thank you!